atomic_reactor/pulp_util.py

```diff
@@ -285,5 +285,5 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         try:
             self.pulp.deleteRepo(repo_id)
         except dockpulp.errors.DockPulpError as exc:
-            logging.info("ignoring error from lock repository deletion: %s",
+            logger.info("ignoring error from lock repository deletion: %s",
                          exc)
```

Comment:
nit: now it doesn't line up.

why not join with the previous line though...

Reply:
Good catch. Fixed it.